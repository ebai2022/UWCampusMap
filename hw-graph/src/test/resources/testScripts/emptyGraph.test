# Test empty graph behavior

# Create a graph
CreateGraph A

# Check if there are any nodes
ListNodes A