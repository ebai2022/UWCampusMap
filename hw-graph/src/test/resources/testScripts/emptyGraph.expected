# Test empty graph behavior

# Create a graph
created graph A

# Check if there are any nodes
A contains: